test/user.test.js

```diff
@@ -0,0 +1,245 @@
+const request = require('supertest');
+const expect = require('chai').expect;
+
+const app = require('../index');
+const fixture = require('../fixtures/user');
+
+describe('User controller', () => {
+  before(async () => {
+    await fixture();
+  });
+
+  describe('GET /api/users/:id', () => {
+    it('#responds with existing user\'s body that has a matching id.', done => {
+      request(app)
+        .get('/api/users/1')
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body.id).to.equal(1);
```

Comment:
Please do not verify only the id, but all the body with all the fields. You can use `expect(res.body).to.deep.equal({ the object here });`.